src/main/java/com/google/devtools/build/lib/buildtool/BuildRequest.java

```diff
@@ -131,6 +132,20 @@ public Builder setCheckforActionConflicts(boolean checkForActionConflicts) {
       return this;
     }
 
+    /**
+     * If true, build status depends on whether or not requested targets are platform-compatible
+     * ({@link com.google.devtools.build.lib.analysis.IncompatiblePlatformProvider}. If false, this
```

Comment:
Nit: You have an open paren and no close paren.

Reply:
Done.